### DISCUSSION TITLE
Implementation Rudelt-Estimator and network inference algorithm

### DISCUSSION THREAD

**Discussion by pwollstadt** [pwollstadt/IDTxl#69] [Ideas]
Hi everyone,
this threads is for discussing the implementation of Lucas' work in IDTxl.

**Comment by mwibral** [comment]
Hi Patricia,
 alles klar. Ich hatte heute morgen schon ein Gespräch mit Michael. Der Original-Code muß ziemlich weitgehend umgebaut werden, da unter anderem auf file-based communication setzt - und damit checkpointing for free bekommt. Wir werden das trennen müssen.
M

**Comment by mwibral** [comment]
uups,
next post will be in English :-)
